papierowyRPG_API/Models/Skills.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Skills
+    {
+        public int ID { get; set; }
+        public string name { get; set; }
+        public string description { get; set; }
```

Comment:
```suggestion
        public string Name { get; set; }
        public string Description { get; set; }
```

Reply:
For the next time, you can just click `Commit suggestion` button to add suggested changes, if you agree with them ;)
Also `Add suggestion to batch` allows you to commit multiple suggested changes in one go